server/controllers/article.js

```diff
@@ -17,17 +15,25 @@ const createArticle = async (req, res) => {
       categoryId
     });
 
-    res.status(201).json({
+    return res.status(201).json({
       status: 'success',
       message: 'Article created successfully',
       data: newArticle
     });
   } catch (error) {
-    res.status(500).json({
+    return res.status(500).json({
       status: 'failed',
-      message: 'Server error'
+      message: error.message
     });
   }
 };
 
-export default createArticle;
+export const getArticles = async (req, res) => {
+  const articles = await Article.findAll();
```

Comment:
we need a where clause, the article must be published before it can be available for the public, also don't forget URL for video and image